package.json

```diff
@@ -53,6 +53,7 @@
     "@types/styled-system": "5.1.10",
     "@types/styled-system__css": "5.0.14",
     "@types/styled-system__theme-get": "5.0.1",
+    "chroma-js": "2.1.0",
```

Comment:
I think chroma-js can be a dev dependency because it's only used in `preval` file that will only be run at build time

Reply:
done!